src/IIIFPresentation/Repository/Repository.csproj

```diff
@@ -12,7 +12,7 @@
         <PackageReference Include="EFCore.NamingConventions" Version="8.0.3" />
         <PackageReference Include="iiif-net" Version="0.2.7" />
         <PackageReference Include="MediatR" Version="12.4.0" />
-        <PackageReference Include="Microsoft.EntityFrameworkCore" Version="8.0.7" />
+        <PackageReference Include="Microsoft.EntityFrameworkCore" Version="8.0.4" />
```

Comment:
Out of curiosity - why downgrade?

Reply:
this was because of the `Npgsql.EntityFrameworkCore.PostgreSQL` package, which has a latest version of `8.0.4` - it was causing warnings to be thrown around package versions